.github/release/full/Dockerfile

```diff
@@ -39,7 +39,7 @@ ENV TERM linux
 
 # Install libgdal.
 RUN apt-get update
-RUN apt-get --yes install libgdal20
+RUN apt-get --yes install libgdal20 gcc g++
```

Comment:
Hi Nico,

doesn't `build-essential` already satisfy the requirements here?

With kind regards,
Andreas.

Reply:
Hi Andreas,

yes you are right! But at the other places I had to add it, otherwise the build crashed.
It seems like there is no WHL file for duckdb and it needs to build from source. That's why it tries to access gcc/g++ which was not available. Now it takes couple of minutes to build from source, not sure if we can fix it without adding gcc/g++.

Kind regards,
Nico

Reply:
Thanks for the insights. 

> Seems it was working before (also on the master) not sure why it is not working now.

We are regularily following DuckDB's releases. Version 0.3.3 was released just five days ago.

I quickly tried to reproduce the build with a minimal `Dockerfile`.
```Dockerfile
FROM python:3.8.8-slim
ENV DEBIAN_FRONTEND noninteractive
ENV TERM linux
RUN pip install wetterdienst[sql,export,restapi]
```

However, it bailed out when aiming to build DuckDB. In abundance to that, it should not do in the first place, because `.whl` packages for 0.3.2 are well available [2], no? The same happens when switching to `python:3.9-slim`. So, I figure there is currently something wrong in this area.

With kind regards,
Andreas.

[1] https://github.com/duckdb/duckdb/releases/tag/v0.3.3
[2] https://pypi.org/project/duckdb/0.3.2/#files

Reply:
I have been able to successfully build an image that way, by leaving out the `sql` extra. The build does not need `build-essential`.
```
FROM python:3.9-slim
ENV DEBIAN_FRONTEND noninteractive
ENV TERM linux
RUN pip install wetterdienst[export,restapi]
```

So, I recommend to just adjust the `standard/Dockerfile` within your build to remove this dependency for now and to be able to move forward with your patch. Please also remove both the `sql` and `duckdb` extras within `full/Dockerfile` as well. We will sort this out later.

Reply:
I think so too that there should be WHL files available but not sure why it tries to build from source. I was able to build the docker image locally by adding gcc and g++ (running the arm64 version of the image).

Reply:
Alright, thanks a lot, I will try out your changes :)

Reply:
> I was able to build the docker image locally by adding gcc and g++ (running the arm64 version of the image).

Ah I see. I tried on amd64 and it croaked while attempting to compile.